### ISSUE TITLE
Rend Blackhand does not glow yellow if the only target is your own Legendary minion

### ISSUE THREAD

**Issue by Patashu** [HearthSim/hs-bugs#89]
Rend Blackhand is missing the yellow glow if the only target for his Battlecry is your own Legendary minion.
https://www.reddit.com/r/hearthstone/comments/3a7c02/is_it_a_rend_blackhand_bug/
https://www.reddit.com/r/hearthstone/comments/3a743q/rend_blackhand_has_no_borders_if_the_enemy_doesnt/
https://www.reddit.com/r/hearthstone/comments/3a6sc7/rend_blackhand_missleading_text/
https://www.reddit.com/r/hearthstone/comments/3aim8u/shouldnt_rend_blackhand_be_glowing_when_you_can/

**Comment by jleclanche** [comment]
This is by design, so closing as invalid.

**Comment by culinko** [comment]
I still think this is a valid bug. Blackwing Corruptor also glows if the only valid targets are friendly.

**Comment by troggnostupidhs** [comment]
Still happens

![hearthstone screenshot 12-20-16 19 40 07](https://cloud.githubusercontent.com/assets/22508641/21373431/edb618fc-c6ec-11e6-8c15-a351241cd357.png)

**Comment by troggnostupidhs** [comment]
If it is intended, they should change the text to say 'destroy enemy Legendary'

**Comment by jleclanche** [comment]
I agree it's inconsistent but the yellow text is most likely meant here as a gameplay helper, not a replica of when the card has its extra power activated.

We can leave open until there's an official word on it... I guess it could be either way.

**Comment by PattuX** [comment]
![image](https://user-images.githubusercontent.com/15743638/33092523-aa177106-cefa-11e7-8528-8984a308e05b.png)

I guess I'll just leave this here, as it's not really worth an own issue and it basically adds to the same "when should something glow yellow"-discussion. Mark of Y'Shaarj only glows yellow if you have a friendly beast, but not if only your opponent has one.

Also Golakka Crawler glows yellow if you have a Pirate, but your opponent doesn't.

**Comment by troggnostupidhs** [comment]
https://www.reddit.com/r/hearthstone/comments/7lfe17/glitch_i_found_in_tavern_brawl/
